src/components/LandingPage.jsx

```diff
@@ -207,7 +219,7 @@ const LandingPage = ({history}) => {
 
                 <>
                     <h1 style={{textAlign: "center", margin: '40px 0px 40px 0px'}}>DOMINO statistics</h1>
-                    <Carousel variant="dark">
+                    <Carousel className="bg-dark">
```

Comment:
I couldn't get this to work! I'm glad you figured out the solution!!!

Reply:
10x :)